src/components/Question.js

```diff
@@ -146,11 +154,13 @@ class Question extends Component {
             label="Previous"
             destination={navigation.previous.url}
             enabled={navigation.previous.enabled}
+            onClick={() => {this.handleCorrectAnswer(null)}}
```

Comment:
Is this necessary? I would think that navigating to a new Question would reinitialize the component state which would automatically set the state to null.

Reply:
That makes sense, I'll switch that out.